draft-ietf-httpbis-digest-headers.md

```diff
@@ -267,8 +270,8 @@ Representation-Digest: \
   iYllu7BNNyealdVLvRwEmTHWXvJwew==:
 ~~~
 
-Since `Representation-Digest` is a Dictionary, it can contain multiple
-members. This could be used, for example, to attach multiple digests
+The `sf-dictionary` syntax, can
```

Comment:
@LPardue this text is now identical in both C-Digest and R-Digest. I guess if there's a way to avoid repetitions